src/main/java/roboy/ros/RosManager.java

```diff
@@ -52,27 +46,26 @@ boolean initialize(ConnectedNode node) {
                     LOGGER.warn("{} client initialization FAILED!", client.toString());
                 }
             }
-
-
         }
 
         for(RosSubscribers subscriber : RosSubscribers.values()) {
-
             if (ConfigManager.ROS_ACTIVE_PKGS.contains(subscriber.rosPackage)) {
                 try {
                     subscriberMap.put(subscriber, node.newSubscriber(subscriber.address, subscriber.type));
                     LOGGER.info("{} subscriber initialization SUCCESS!", subscriber.toString());
-                } catch (Exception e) {
+                }
+                catch (Exception e) {
```

Comment:
the old code style (catch and braces in one line) is preferred, see http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html (point 7.4)